README.md

```diff
@@ -38,7 +38,7 @@ Please refer to the [installation guide](https://chainermn.readthedocs.io/en/lat
 You can invoke MNIST example with four workers by the following command:
 
 ```
-mpiexec -n 4 python examples/mnist/train_mnist.py
+mpiexec -n 4 python examples/mnist/train_mnist_data_parallel.py
```

Comment:
I thought it's confusing if `train_mnist.py` and `train_mnist_model_parallel.py` exist at the same time, so renamed the original `train_mnist.py` to `train_mnist_data_parallel.py`, in order to explicitly assert this example is for data parallel.